app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -20,9 +20,7 @@ module AuthnJwt
     EXP_CLAIM_NAME = "exp"
     NBF_CLAIM_NAME = "nbf"
     IAT_CLAIM_NAME = "iat"
-    RSA_ALGORITHMS = %w[RS256 RS384 RS512].freeze
-    ECDSA_ALGORITHMS = %w[ES256 ES384 ES512].freeze
-    SUPPORTED_ALGORITHMS = (RSA_ALGORITHMS + ECDSA_ALGORITHMS)
+    SUPPORTED_ALGORITHMS = %w[RS256 RS384 RS512].freeze
```

Comment:
Maybe this algorithm change should be documented in changelog

Reply:
As far as I understand jwt documentation yet published and in the version that will be published will be no ES* algs.
Do you thin that in this case we still need change log entity for the change?

Reply:
Dont know.
@hilagross decide

Reply:
we are documenting that it is not supported.
I'm thinking maybe we should remove that comment as well and not mention it at all.
I don't think we need a change log, this is more like fixing a bug as this is not out yet.